### PR TITLE
Keyboard: XeaL60

### PR DESCRIPTION
**Pull request by alex-ong**
Request to add my own personal split keyboard with it's own debouncing algorithm.

### CODE REVIEW

**Comment by alex-ong** [comment]
I'm not using the shared split_util stuff because i've changed the debouncing algorithm.
Plan for now is to use this code until shared debounce stuff gets merged into master, then i'll make another pull request for shared debounce code inside qmk/split_common, and then finally all these files will disappear.

File: keyboards/handwired/xeal60/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+   * |-----------------------------------------------------------|
+   * |FN     |  A|  S|  D|  F|  G|  H|  J|  K|  L|  ;|  '|Return |
+   * |-----------------------------------------------------------|
+   * |Shift   |  Z|  X|  C|  V|  B|  N|  M|  ,|  .|  /| Shift    |
+   * |-----------------------------------------------------------|
+   * |Ctrl|Gui |Alt | NUM | Space |  Space |Alt |FN  |Menu |Ctrl |
+   * `-----------------------------------------------------------'
+   */
+  /* Layer 0: Qwerty */
+  [_QWERTY] = LAYOUT_split60( \
```
[the remark this answers is not in the document]
Comment (alex-ong) [reply keyboards/handwired/xeal60/keymaps/default/keymap.c]: It's got an extra key (numpad layer), so it's not exactly the same. Is there a way to include community layouts feature while also adding a single key?

**Comment by alex-ong** [comment]
Made all the changes (i think). Renamed keyboard to xealous, hopefully that is better?

**Comment by drashna** [comment]
Okay, I had a feeling that was the case.  And that's fine.  

And yeah, I think that name is on the better side.

File: keyboards/handwired/xealous/keymaps/default/config.h [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "../../config.h"
```
Comment (drashna) [inline keyboards/handwired/xealous/keymaps/default/config.h]: This line shouldn't be needed, at all.  The make system actually does this automatically, so no need to include it.

**Comment by Wilba6582** [comment]
I wouldn't have cared about "XeaL60", but I was thinking of calling a future custom keyboard "Xealous" lol

**Comment by alex-ong** [comment]
Removed one of the config.h's. 

I think i've done all the changes you requested :octocat:

File: keyboards/handwired/xealous/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,105 @@
+#include QMK_KEYBOARD_H
+#include "config.h"
```
Comment (drashna) [inline keyboards/handwired/xealous/keymaps/default/keymap.c]: This isn't needed.  It's included by the make system automatically

Comment (alex-ong) [reply keyboards/handwired/xealous/keymaps/default/keymap.c]: Which ones?
When i remove both, it doesn't build

Comment (alex-ong) [reply keyboards/handwired/xealous/keymaps/default/keymap.c]: It builds when i remove config.h, but not if i remove 
```
#include QMK_KEYBOARD_H
```

**Comment by drashna** [comment]
Awesome, thanks!

File: keyboards/handwired/xealous/keymaps/default/keymap.c (hunk 3 above)
Comment (drashna) [reply keyboards/handwired/xealous/keymaps/default/keymap.c]: Yeah, you need the keyboard on there.  The config.h is actually automatically included by the make script, IIRC.  So you don't need that included.

**Pull request merged**